src/constants/manifest.consts.ts

```diff
@@ -0,0 +1,29 @@
+import { MetadataRoute } from 'next'
+
+export const manifest: MetadataRoute.Manifest = {
+    name: 'Peanut Protocol',
+    short_name: 'Peanut',
+    description: 'Buttery smooth global money',
+    icons: [
+        {
+            src: '/pwa/icon-192x192.png',
+            sizes: '192x192',
+            type: 'image/png',
+        },
```

Comment:
Q: isn't this kinda duplicate with sw.ts?

Reply:
Do you mean manifest.ts ?